aoreports.civix.php

```diff
@@ -6,10 +6,10 @@
  * The ExtensionUtil class provides small stubs for accessing resources of this
  * extension.
  */
-class CRM_Aoreports_ExtensionUtil {
-  const SHORT_NAME = "aoreports";
-  const LONG_NAME = "biz.jmaconsulting.aoreports";
-  const CLASS_PREFIX = "CRM_Aoreports";
+class CRM_AOReports_ExtensionUtil {
```

Comment:
@monishdeb the casing here might be suggestive of a problem perhaps?

Reply:
@seamuslee001 updated the PR. Wonder how the report classes are correctly declaring the extension util class here. May the fix was done partially!!

Reply:
grepped the occurrences after the change:
```
Monishs-MacBook-Pro:biz.jmaconsulting.aoreports monish$ grep -irn "CRM_AOReports_ExtensionUtil" .
./CRM/AOReports/Form/Report/ExtendedActivity.php:2:use CRM_AOReports_ExtensionUtil as E;
./CRM/AOReports/Form/Report/ExtendContributionDetails.php:2:use CRM_AOReports_ExtensionUtil as E;
./CRM/AOReports/Form/Report/SNPWAITDAY.php:2:use CRM_AOReports_ExtensionUtil as E;
./CRM/AOReports/Form/Report/ExtendSummary.php:2:use CRM_AOReports_ExtensionUtil as E;
./CRM/AOReports/Form/Report/EventsThisQuarter.php:2:use CRM_AOReports_ExtensionUtil as E;
./aoreports.civix.php:9:class CRM_AOReports_ExtensionUtil {
./aoreports.civix.php:80:use CRM_AOReports_ExtensionUtil as E;
./aoreports.php:4:use CRM_AOReports_ExtensionUtil as E;
```
seems ok to me.